apple/MarkdownTextInputDecoratorView.mm

```diff
@@ -6,59 +6,34 @@
 #import <RNLiveMarkdown/RCTBackedTextFieldDelegateAdapter+Markdown.h>
 #import <RNLiveMarkdown/RCTUITextView+Markdown.h>
 
-#ifdef RCT_NEW_ARCH_ENABLED
 #import <RNLiveMarkdown/RCTTextInputComponentView+Markdown.h>
-#else
-#import <RNLiveMarkdown/RCTBaseTextInputView+Markdown.h>
-#endif /* RCT_NEW_ARCH_ENABLED */
 
 #import <objc/runtime.h>
 
 @implementation MarkdownTextInputDecoratorView {
   RCTMarkdownUtils *_markdownUtils;
   RCTMarkdownStyle *_markdownStyle;
   NSNumber *_parserId;
-#ifdef RCT_NEW_ARCH_ENABLED
   __weak RCTTextInputComponentView *_textInput;
-#else
-  __weak RCTBaseTextInputView *_textInput;
-#endif /* RCT_NEW_ARCH_ENABLED */
   __weak UIView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
   __weak RCTBackedTextFieldDelegateAdapter *_adapter;
   __weak RCTUITextView *_textView;
 }
 
 - (void)didMoveToWindow {
-#ifdef RCT_NEW_ARCH_ENABLED
   if (self.superview.superview == nil) {
     return;
   }
```

Comment:
It should be possible to merge Decorator view and Decorator component view now (in a follow-up, of course)

Reply:
What exact changes do you suggest?

Reply:
I thought that `MarkdownTextInputDecoratorComponentView` was a wrapper around `MarkdownTextInputDecoratorView` but now that I've checked it looks like two independent implementations.

Is `MarkdownTextInputDecoratorView` used at all with these changes?

Reply:
Let me check that